client/src/components/imagine/components/Bias.jsx

```diff
@@ -14,7 +14,7 @@ const Bias = (props) => {
     isModalActive,
     setModalActive,
   } = props;
-  const [setTimerDone] = useState(false);
+  const [TimerDone, setTimerDone] = useState(false);
```

Comment:
This is a nitpick but TimerDone should probably be timerDone (camel case)

Reply:
Ok, I'll change that